CHANGELOG.rst

```diff
@@ -5,6 +5,17 @@ Community DigitalOcean Release Notes
 .. contents:: Topics
 
 
+v1.1.0
```

Comment:
This change should NOT have been in this PR, and neither the change for changelog.yaml.

These files must ONLY be updated on release.